tweets.sql

```diff
@@ -0,0 +1,36 @@
+-- USERDATA table for holding the user information (username, user full name, and user id)
+CREATE TABLE USERDATA(
+    userid NUMBER NOT NULL,
+    username VARCHAR2(255 char) NOT NULL,
+    userfullname VARCHAR2(255 char),
+    PRIMARY KEY (userid)
+);
+-- sequence counter for the unique user id, starts from 1 and increment by 1
+CREATE SEQUENCE seq_userid
+MINVALUE 1
+START WITH 1
+INCREMENT BY 1
+CACHE 10;
+
+--INSERT INTO USERDATA (userid)
+--VALUES (seq_userid.nextval);
+
+-- TWEETDATA table holds the tweet information, (message, tweet id, title, mode)
+CREATE TABLE TWEETDATA(
+    userid NUMBER,
+    id NUMBER  NOT NULL,
```

Comment:
According to you [swagger definition](https://github.com/osu-mist/tweet-api/blob/master/swagger.yaml#L165), the tweet ID should be a string. Same with your user ID, even user ID won't be exposed to the client.